content/en/apps/features/integrations/oppia.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "OppiaMobile"
+weight: 3
+description: >
+   Integrate CHT core with OppiaMobile’s learning management platform
+keywords: oppia
+relatedContent: >
+  apps/examples/learning-care
+  apps/guides/integrations/oppia
+---
+
+[OppiaMobile](https://digital-campus.org/oppiamobile) is an open source mobile learning platform built by Digital Campus specially designed for delivering learning content, multimedia, and quizzes in low-broadband settings. All the content and activities can be accessed and used when no internet connection is available, and users can earn points and badges for completing activities and watching videos. To learn more about the platform, check out the [overview](https://digital-campus.org/oppiamobile/developers), OppiaMobile on [Github](https://github.com/DigitalCampus), and their [documentation site](http://oppiamobile.readthedocs.io/en/latest). You can also join the OppiaMobile [Community Discussion Board](https://community.oppia-mobile.org).
```

Comment:
TBH i had to look up what `low-broadband` meant, I'd never heard it before. What words do we use to describe this elsewhere?

Reply:
that is language lifted from Oppia materials, we can change to `low connectivity settings`